src/core/log.js

```diff
@@ -18,7 +19,11 @@ if (sendLogs && sendLogsTo) {
 }
 
 streams.push(
-    {level: options.level, stream: pino({prettyPrint: {colorize: true}})[pino.symbols.streamSym]},
+    {
+        level: options.level,
+        stream: pino({prettyPrint: {colorize: true}})[pino.symbols.streamSym],
+        address: getNetworkAddress()
```

Comment:
This doesn't add address to each log message like we need. Instead we should do something like that:
```
streams.push(...);

let logger = pino(options, multistream(streams));

try {
  const account = getNetworkAddress();
  logger = logger.child({account});
} catch (e) {
  // no account address
}

module.exports = Object.assign(logger, {
  ....
});
```